PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/PrebidServerAdapter.java

```diff
@@ -306,6 +307,13 @@ protected void onPostExecute(AsyncTaskResult<JSONObject> asyncTaskResult) {
                                                 String key = (String) it.next();
                                                 keywords.put(key, hb_key_values.getString(key));
                                             }
+                                            String cacheId = CacheManager.save(bid.toString());
```

Comment:
Should we save only for native ad request to save some memory space?

Reply:
Done